src/components/FeatureSection.jsx

```diff
@@ -1,39 +1,92 @@
-import * as React from 'react'
+import * as React from "react";
+import {
```

Comment:
Please use react icons, same comment is given to Faraz as well, please use a single lib and levarage icons from that